### ISSUE TITLE
Add Settings Controller

### ISSUE THREAD

**Issue by drop-table-lol** [uidaho/squireproject#89]

**Comment by rickbau5** [comment]
Note that we need the basic of this for the sprint ending on 4/25, see the [upcoming milestone](https://github.com/uidaho/squireproject/milestones/Foundation%20&%20Tests)

**Comment by rickbau5** [comment]
@drop-table-lol

**Comment by drop-table-lol** [comment]
Yeah, Eric and I are doing a session tomorrow after class, and probably again on friday

**Comment by rickbau5** [comment]
Alright, well we need something merged into master by Sunday night just giving you guys a heads up.

**Comment by rickbau5** [comment]
These are the use cases described in C9 for settings, this can be expanded to their own issues once settings is implemented 
- Modify chat font Settings 
- Modify chat color Settings 
- View Line Numbers
- View References 
- View Dates
- View Authors
- View A User’s Profile 
- Modify Profile Info
- Modify Profile Privacy

@drop-table-lol @drspaceman0
